larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -654,6 +682,10 @@ StatusCode MasterAlgorithm::Copy(const Pandora *const pPandora, const CaloHit *c
     }
     LArCaloHitParameters parameters;
     pLArCaloHit->FillParameters(parameters);
+
+    if (ignoredHit)
+        parameters.m_nCellInteractionLengths = -999.f;
+
```

Comment:
This is the core of why this is implemented in a very hacky way right now...

I want to pass over every single calo hit, but I don't want the downstream algorithms to actually use all these hits, I just want them to carry on as normal.

My "fix" was to hijack a seemingly unused var and use that to flag in the pre-processing "Hey this is a hit you should not actually use, just store it to one side".